package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Jul 12 05:37:30 UTC 2017 - mfilka@suse.com
+
+- bnc#1038717
+  - avoid creating duplicit udev rules in AutoYaST installation
```

Comment:
"duplicit" is not a word. Please consider https://twitter.com/mvidner/status/884388828701700096
- http://www.thefreedictionary.com/duplicity
- http://www.thefreedictionary.com/duplicate